lib/security/security-spec.js

```diff
@@ -0,0 +1,70 @@
+var assert = require('assert');
+var sampleUserProfileData = require('../../test/fixtures/sampleUserProfile.json');
+var aes = require('./aes.js');
+var sha256 = require('./sha256.js');
+var sampleSecurityeData = require('../../test/fixtures/sampleSecurityData.json');
+
+describe("Test Data Encyption, Hashing and Decryption", function() {
+  it('should error when you try to hash a null value', function(done) {
+    return sha256.hash(null).then(function(response) {
+      assert(!response, "Should not contain a hash value,");
+    }).then(done).catch(function(err) {
```

Comment:
Also, if you use mocha and don't inject _done_, but do _return_ the promise it can pass/fail depending on if the Promise is rejected/resolved

```
it('will fail since it rejects', function () {
  return encrypt('BADPARAMS', null);
});

it('will pass since it resolves', function () {
  return encrypt('GOODPARAMS', 'MORE GOOD PARAMS')
    .then((encrypted) => {
      // can even do checks here
      assert.equal(typeof encrypted, 'string');
    });
});
```